deps/ledger-zxlib/src/zbuffer.c

```diff
@@ -54,7 +54,7 @@ zbuffer_error_e zb_allocate(uint16_t size) {
     zbuffer_internal.ptr = (uint8_t * )(&app_stack_canary + 4);
 
     uint32_t *zb_canary = (uint32_t * )(zbuffer_internal.ptr + zbuffer_internal.size + 4);
-    cx_rng((unsigned char *)&CANARY_EXPECTED_RANDOM, 4);
+    cx_rng_no_throw((unsigned char *)&CANARY_EXPECTED_RANDOM, 4);
```

Comment:
Return value must be checked here.

Reply:
I am agree, but this function is define on include as a void function:
https://github.com/LedgerHQ/ledger-nanos-sdk/blob/master/nanos-secure-sdk/lib_cxng/include/lcx_rng.h#L38